README.md

```diff
@@ -221,6 +238,17 @@ You can run the checks yourself using:
 pre-commit run --all-files
 ```
 
+#### 6. Tagging of changes
+In order to have release notes automatically generated, changes need to be tagged with labels.
+The following labels are used (should be safe-explanatory):
+`breaking-change`, `bug`, `enhancement`.
```

Comment:
Shall we add these in issues or commit messages?

Reply:
I think PR is the best, then it ends up in the release notes.

Reply:
Sounds good